### PR TITLE
Set ticker in onConfigure instead of onStart

### PR DESCRIPTION
**Pull request by bianpengyuan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by bianpengyuan** [comment]
This could be the cause of https://github.com/istio/istio/issues/21505

**Comment by bianpengyuan** [comment]
@kyessenov @jplevyak @douglas-reid fyi

File: extensions/stackdriver/stackdriver.cc [hunk 1]
```diff
@@ -189,6 +189,12 @@ bool StackdriverRootContext::onConfigure(
 
   node_info_cache_.setMaxCacheSize(config_.max_peer_cache_size());
 
+  // All the necessary reporting objects should be initialized.
```
Comment (douglas-reid) [inline extensions/stackdriver/stackdriver.cc]: This needs to move to the top of `onConfigure`.

Comment (douglas-reid) [reply extensions/stackdriver/stackdriver.cc]: It should match the `release-1.5` changes:

https://github.com/istio/proxy/blob/master/extensions/stackdriver/stackdriver.cc#L140-L146.

And we should probably match `master`.

Comment (douglas-reid) [reply extensions/stackdriver/stackdriver.cc]: OK. we are going to update 1.5 to match this placement.

**Pull request merged**